### PR TITLE
Hover effects modified for nav

### LINKED ISSUE #122: Enhancement
**Describe the bug**
Light blue color on hover on Nab Option is not matching the website Theme
**To Reproduce**
- This should be replace with Light pink Color same as in Illustrations

### PR DESCRIPTION
**Pull request by nishihere19**
#### Description
Issue FIXES #122 

#### Describe the changes you've made

Changed the color of the hover effect for enhancement of the navbar

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)

NA.
#### Additional context (OPTIONAL)

Screenshot from 2021-03-23 11-55-23
Screenshot from 2021-03-23 11-55-26

#### Checklist

- [ ] I have added my name in the contributors list at the end of README.md file.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [x] Open Source Program names: GSSOC'21

### CODE REVIEW

File: periodo/.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-AUTH0_DOMAIN=
-AUTH0_SECRET=
-AUTH0_CLIENT_ID=
-COOKIE_SECRET=
\ No newline at end of file
+AUTH0_DOMAIN=dev-jepc3b7g.us.auth0.com
```
Comment (Aaishpra) [inline periodo/.env:1]: Remove this file, Never commit your secrets on github

Comment (nishihere19) [reply periodo/.env:1]: sure

**Review by Aaishpra — APPROVED** [review]
Lgtm, thanks for the contribution @nishihere19

**Comment by Aaishpra** [comment]
@nishihere19 can you remove the conflicts then i can merge this

**Comment by nishihere19** [comment]
Done. :smiley:

**Pull request merged**